h2o-genmodel/src/test/java/hex/genmodel/ModelMojoFactoryTest.java

```diff
@@ -0,0 +1,19 @@
+package hex.genmodel;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class ModelMojoFactoryTest {
+
+  @Rule
+  public ExpectedException thrown = ExpectedException.none();
+
+  @Test
+  public void getUnknownMojoReader() throws Exception {
+    thrown.expectMessage("Algorithm `Unknown` is not supported by this version of h2o-genmodel. " +
+            "If you are using algorithm implemented in an extension be sure to include a jar dependency of the extension (eg.: ai.h2o:h2o-genmodel-ext-unknown)");
```

Comment:
Updated string:
"If you are using an algorithm implemented in an extension, be sure to include a jar dependency of the extension (eg.: ai.h2o:h2o-genmodel-ext-unknown)"); 

Also, do you not need + algo.toLowerCase() + ")") in this string?

Reply:
"+ algo.toLowerCase()" is supposed to complete the partial maven artifact coordinates "ai.h2o:h2o-genmodel-ext-"; I think it gives the user more guidance. WDYT?

Reply:
I just noticed that algo.toLowerCase() + ")") was used in the string in the ModelMojoFactory.java file but not in the ModelMojoFactoryTest.java file. I wasn't sure if there was a specific reason why the strings would be different.

Reply:
I understand now. The test already has hardcoded "unknown" (instead of Unknown) - this way it is also testing that the message will be generated properly.

Reply:
Ahh, okay. Then just those couple of requested changes:
"If you are using algorithm" > "If you are using an algorithm"
and a comma before "be":
"... implemented in an extensions, be sure to ..."